pkg/vpcmodel/output.go

```diff
@@ -215,15 +209,17 @@ func (of *serialOutputFormatter) WriteOutput(c1, c2 MultipleVPCConfigs, conns ma
 		}
 		return of.AggregateVPCsOutput(outputPerVPC, uc, outFile)
 	}
-	// its diff or explain mode, we have only one vpc on each map:
-	name, _ := common.AnyMapEntry(c1)
-	vpcAnalysisOutput, err2 :=
-		of.createSingleVpcFormatter().WriteOutput(c1[name], c2[name], conns[name], subnetsConns[name],
-			subnetsDiff, "", grouping, uc, explainStruct)
-	if err2 != nil {
-		return "", err2
+	for name, vpcConfig := range cConfigs.Configs() {
+		vpcAnalysisOutput, err2 :=
+			of.createSingleVpcFormatter().WriteOutput(vpcConfig, cConfigs.ConfigToCompare(name), conns[name], subnetsConns[name],
```

Comment:
Wouldn't it make sense to change the interface of the relevant `WriteOutput` instead of passing both `vpcConfig` and `cConfigs.ConfigToCompare(name)`

Reply:
here .WriteOutput() gets two vpcConfigs (and not two MultipleVPCConfigs).
the receiver is a SingleVpcFormatter, it does not know multi vpcs. 
you can suggest another interface, I think that this one is OK.
however, it is outside this PR